opnode/node/api.go

```diff
@@ -45,17 +46,22 @@ func (n *nodeAPI) OutputAtBlock(ctx context.Context, number rpc.BlockNumber) ([]
 		return nil, ethereum.NotFound
 	}
 
-	storageHash, err := n.client.GetProof(ctx, n.withdrawalContractAddr, toBlockNumArg(number))
+	accountRoot, proof, err := n.client.GetProof(ctx, n.withdrawalContractAddr, toBlockNumArg(number))
 	if err != nil {
 		n.log.Error("failed to get contract proof", "err", err)
 		return nil, err
 	}
-	if storageHash == nil {
+	if accountRoot == nil {
```

Comment:
You're still returning the storage hash in the `(c *l2EthClientImpl) GetProof` implementation. The account-proof will fail because of that.

Reply:
fixed it